callmetadata.schema.json

```diff
@@ -5,29 +5,87 @@
     "description": "Meta data supporting dialtone, ringdown, and shoutdown calls",
     "type": "object",
     "definitions": {
-        "participant": {
+        "states": {
             "type": "object",
-            "description": "Specific details about the farEndUsers (the other end of the call)",
             "properties": {
-                "firmName": {
+                "state": {
                     "type": "string",
-                    "example": "XYZ Firm",
-                    "description": "Name of the farEnd firm on the call. If an internal call, the firmName can match the nearEnd firm"
+                    "description": "name of the latest state of the current communication: CallOriginated, CallDelivered, CallEstablished, ect. Could be an enum with defined states. ",
+                    "example": "CallDelivered"
                 },
-                "groupId": {
+                "time": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "Date and time when the state changed for the given participant",
+                    "example": "2016-06-18T08:05:15Z"
+                },
+                "duration": {
                     "type": "integer",
-                    "example": 235,
-                    "description": "The internal ID used by consumer for the farEnd group. This will never change for the connection"
+                    "description": "Time in milliseconds how of how long the call has been in previous states for the given participant",
+                    "example": 27978
+                }
+            }
+        },
+        "member": {
```

Comment:
@pinczo What's the reason to move these fields to another definition(member) instead of keeping it in participant? 

Thank you

Reply:
Participant is about who was on the call and the details of their participation - IE time, states etc. It includes member as an element. Member is more about the person themselves and under what guise they are active on the call. We can then re-use member as a definition elsewhere.